example/site.js

```diff
@@ -55,4 +55,17 @@ const PreserveFormPlugin = (element) =>{
     });
     localStorage.setItem(key, JSON.stringify(values));
   }
+}
+
+document.addEventListener("pill:ready", appendEvent);
+document.addEventListener("pill:mounting", appendEvent);
+document.addEventListener("pill:unmounting", appendEvent);
+document.addEventListener("pill:loading", appendEvent);
+document.addEventListener("pill:shouldserve", appendEvent);
+document.addEventListener("pill:shouldreload", appendEvent);
+document.addEventListener("pill:error", appendEvent);
+document.addEventListener("pill:fromerror", appendEvent)
+
+function appendEvent(event){
+  console.info(event);
 }
```

Comment:
Pill doesn't use semicolons and requires trailing empty line in the end of the file, but it wasn't forced. 

I've added lint rules and ESLint as dependency. But haven't added CI/CD to check this automatically. To fix issues please run `npx eslint example`.

Reply:
Can we let this slide?
  I think we should add a rollup plugin to strip out things like semicolons, etc.  We can also add dev debug messages and have them stripped out in a prod build

Reply:
This rule is for the code of sources, not for production code. This is why it should be made by developer. You can install project dependencies from branch v1.5 and run `npm run lint:fix` to fix semicolon issues automatically.

Reply:
Will do.